runs/run_taylor_fwd_inv.py

```diff
@@ -0,0 +1,99 @@
+# For fenics_ice copyright information see ACKNOWLEDGEMENTS in the fenics_ice
+# root directory
+
+# This file is part of fenics_ice.
+#
+# fenics_ice is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# fenics_ice is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with tlm_adjoint.  If not, see <https://www.gnu.org/licenses/>.
+
+# This script does not output anything. It is only use for
+# running taylor test verification on the forward runs outside
+# of the pytest environment
+
+import os
+os.environ["OMP_NUM_THREADS"] = "1"
+os.environ["OPENBLAS_NUM_THREADS"] = "1"
+
+import sys
+
+from runs import run_forward
+
+from fenics_ice.backend import clear_caches, compute_gradient, \
+    reset_manager, stop_manager, taylor_test
+
+def EQReset():
+    """Take care of tlm_adjoint EquationManager"""
+    # This prevents checkpointing errors when these run phases
+    # are tested after the stuff in test_model.py
+    reset_manager("memory")
+    clear_caches()
+    stop_manager()
+
+def test_tv_run_forward(config_file):
+
+    EQReset()
+
+    mdl_out = run_forward.run_forward(config_file)
+
+    slvr = mdl_out.solvers[0]
+
+    qoi_func = slvr.get_qoi_func()
+    cntrl = slvr.get_control()
+
+    slvr.reset_ts_zero()
+    J = slvr.timestep(adjoint_flag=1, qoi_func=qoi_func)[0]

```

Comment:
This tests the first functional, which may be the initial functional, and I don't think this depends on `alpha`

Reply:
same as above